packages/kbn-generate-csv/src/generate_csv.ts

```diff
@@ -355,7 +356,10 @@ export class CsvGenerator {
       ),
       this.dependencies.searchSourceStart.create(this.job.searchSource),
     ]);
-    let reportingError: undefined | ReportingError;
+
+    this.logger.debug(
+      `Task started at: ${this.taskInstanceFields.startedAt}. Available until: ${this.taskInstanceFields.retryAt}`
```

Comment:
It's tough to come up with a clear concise debug message here. 

Maybe this instead?
```suggestion
      `Task started at: ${this.taskInstanceFields.startedAt}. Can run until: ${this.taskInstanceFields.retryAt}`
```